### DISCUSSION TITLE
How to make an element represent a grain?

### DISCUSSION THREAD

**Discussion by louis96108** [neperfepx/neper#893] [Q&A] [answered]
Hi everyone，
I've got a polycrystalline model with a grain shape like a cube, how do I mesh it so that one element corresponds to one grain?
![png](https://github.com/user-attachments/assets/f31164a9-afcc-474c-9b43-0583859b2c44)
Thanks a lot for your reply.

Sincerely,
Louis

**Comment by Abaqus-Voronoi-toolbox** [comment, marked as answer]
I have used it in my paper: Zhou, S., Ben Bettaieb, M., Abed-Meraim, F., 2024b. Investigation of the effect of morphological and crystallographic textures on the ductility limits of thin metal sheets using a CPFEM-based approach. European Journal of Mechanics - A/Solids 106. doi:https://doi.org/10.1016/j.euromechsol.2024.105293.

you can use the following commands:
neper -T -n "from_morpho" -morpho "cube(5,5,5)" -domain "cube(1,1,1)" -o xxx
neper -M xxx.tess -elttype hex -rcl 5

**Reply by rquey** [reply]
I'd recommend using `-cl` instead of `-rcl`:
```
neper -T -n from_morpho -morpho "cube(5)" -o xxx
neper -M xxx.tess -elttype hex -cl 1/5
```
